drivers/backup/portworx/portworx.go

```diff
@@ -720,6 +722,49 @@ func (p *portworx) WaitForRunning(
 	return nil
 }
 
+func (p *portworx) ActivateLicense(req *api.LicenseActivateRequest) (*api.LicenseActivateResponse, error) {
+	return p.licenseManager.Activate(context.Background(), req)
+}
+
+func (p *portworx) InspectLicense(req *api.LicenseInspectRequest) (*api.LicenseInspectResponse, error) {
+	return p.licenseManager.Inspect(context.Background(), req)
+}
+
+func (p *portworx) WaitForLicenseActivation(ctx context.Context, req *api.LicenseInspectRequest, timeout, retryInterval time.Duration) error {
+	var licenseErr error
+
+	t := func() (interface{}, bool, error) {
+		resp, err := p.licenseManager.Inspect(ctx, req)
+
+		if err != nil {
+			return nil, true, err
+		}
+
+		// Check if we got response from license server
+		if len(resp.GetResponseInfo()) == 0 {
+			licenseErr = fmt.Errorf("failed to activate license for orgID %v", req.GetOrgId())
+			return nil, false, licenseErr
+		}
+		// iterate over the feature and check if valid feature is present
+		for _, ft := range resp.GetResponseInfo() {
+			if ft.GetName() != "ClusterCount" {
```

Comment:
It is better to make "ClusterCount" as const.

Reply:
Fixed.